pkg/plugin/plugin.go

```diff
@@ -395,6 +395,9 @@ func CmdAdd(args *skel.CmdArgs) error {
 		// General Case
 		hostIface, contIface, err = setupVeth(contNetns, args.IfName, mac, netconf.MTU)
 	}
+	if err != nil {
```

Comment:
Would you move this right bellow both of the lines catching the error? Even though it would duplicate it, I find it better than having to figure out the whole if else above, plus the global err.